common/OpTestUtil.py

```diff
@@ -2004,7 +2004,10 @@ def __gather_os_files_logs(self, list_of_files=[], output_dir=None):
             host.host_run_command("mkdir -p files")
             for file in set(list_of_files):
                 fn = "%s.log" % '-'.join(file.strip(os.path.sep).split(os.path.sep))
-                host.host_run_command("cp %s files/%s" % (file, fn))
+                try:
+                    host.host_run_command("cp %s files/%s" % (file, fn))
```

Comment:
i feel better way is like check file is there as try/expect not a proper way

Reply:
should we check for all the files which are part of the log folder ?

Reply:
yes , checking should not harm any thing